python/python/ert_gui/simulation/models/single_test_run.py

```diff
@@ -3,15 +3,37 @@
 from res.enkf import ErtRunContext
 from ert_gui.simulation.models import BaseRunModel, ErtRunError, EnsembleExperiment
 
-class SingleTestRun(EnsembleExperiment):
+class SingleTestRun(BaseRunModel):
```

Comment:
Sure this wasn't done on purpose? Just to reuse the runSimulations function from EnsembleExperiment, only changing the mask?